src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -69,27 +69,42 @@ def validate_location(cmd, location):
             break
 
 
-def get_chart_path(registry_path, kube_config, kube_context, helm_client_location):
+def get_chart_path(registry_path, kube_config, kube_context, helm_client_location, for_preonboarding_checks=False):
```

Comment:
instead of this, the new parameter should be "chart_path_name" and then set it accordingly while calling the get_chart_path method

Reply:
default value can be set to AzureArcCharts